tests/helpers/reporter/reporter.go

```diff
@@ -0,0 +1,211 @@
+// Copyright (c) 2016 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"github.com/01org/cc-oci-runtime/tests/helpers/reporter/metric"
+)
+
+var LogPath string
+
+func writeJSONEnvironment() {
+	const JSONFile = "environment"
+	environment := metric.GetEnvironment()
+	environment.WriteJSONEnvironment(JSONFile)
+}
+
+func writeTestResult() {
+
+	//Write JSON for Docker Test Results
+
+	dockerTestRegex := metric.TestRegex{
+		LogFile: LogPath + "docker-integration-tests.html",
+		//		JSONFile:  "docker_test_result.json",
+		SkipField: "# skip",
+		FailField: ">not ok ",
+		PassField: ">ok ",
+	}
+
+	dockerTestResult := metric.Test{
+		Name:        "docker test result",
+		Description: "docker test result for cc-oci-runtime",
```

Comment:
I'd consider dropping mention of `cc-oci-runtime` here and below or if we really think we need it, define a variable for the name (in case we change it again ;-)